packages/core/package.json

```diff
@@ -8,9 +8,10 @@
     "depcheck": "depcheck",
     "pretest": "npm run lint && npm run depcheck",
     "redis": "docker rm --force redis-commerce-sdk; docker run -dp 6379:6379 --name redis-commerce-sdk redis:5-alpine",
-    "redis:shutdown": "docker rm --force redis-commerce-sdk; docker rm --force redis-commerce-sdk-debug",
+    "redis:shutdown": "docker rm --force redis-commerce-sdk",
     "redis:debug": "docker rm --force redis-commerce-sdk-debug; docker run -dp 6379:6379 --name redis-commerce-sdk-debug redis:5-alpine && docker exec -it redis-commerce-sdk-debug redis-cli monitor",
-    "test": "nyc mocha \"test/**/*.test.ts\" --reporter=xunit --reporter-options output=../../reports/core.xml",
+    "redis:debug:shutdown": "docker rm --force redis-commerce-sdk-debug",
+    "test": "nyc mocha \"test/**/*.test.ts\" --reporter=xunit --reporter-options output=../../reports_integration/core.xml",
```

Comment:
Why was this moved?

Reply:
It wasn't moved, a new line was inserted to support redis debug shutdown and the name of the target report folder was also updated (as it's not integration tests). Which makes it look like the line moved.

Reply:
Sorry what i mean is 
```
    "test": "nyc mocha \"test/**/*.test.ts\" --reporter=xunit --reporter-options output=../../reports/core.xml",
```
became
```
    "test": "nyc mocha \"test/**/*.test.ts\" --reporter=xunit --reporter-options output=../../reports_integration/core.xml",
```

Which moved the output of the report.

Reply:
because this is no longer an integration test..

Reply:
Yep, sorry, this was left over from when we were going to put integration tests in code. Resolved it now.